### ISSUE TITLE
[Feature Request] Forward React from Whatsapp

### ISSUE THREAD

**Issue by CottonWhiteSheep** [akshettrj/watgbridge#50]
Able to send message or react to telegram when receiver react to messages on WhatsApp

**Comment by akshettrj** [comment]
I think it would be feasible to only send reactions on your own messages.

Otherwise it can cause a lot of spam

Any thoughts?

**Comment by CottonWhiteSheep** [comment]
Or is it possible to make it optional in the config file?

Or straight react the message on Telegram instead of sending it as a new message to prevent spam (Not sure if it will help on reduce spam on the message side)

The idea behind this is because sometimes people just react instead of replying in chat, so it could be a way to know that someone has seen the chat and respond to it

**Comment by akshettrj** [comment]
Well, never mind, Telegram just dropped Bot API 7.0 which supports full reactions for bot

gg

**Comment by CottonWhiteSheep** [comment]
Yup sounds like a good solutions :) Thank you so much for the help sir
